openmmml/models/anipotential.py

```diff
@@ -81,8 +91,27 @@ def addForces(self,
         includedAtoms = list(topology.atoms())
         if atoms is not None:
             includedAtoms = [includedAtoms[i] for i in atoms]
-        elements = [atom.element.symbol for atom in includedAtoms]
-        species = model.species_to_tensor(elements).unsqueeze(0)
+        # elements = [atom.element.symbol for atom in includedAtoms]
+        #species = model.species_to_tensor(elements).unsqueeze(0)
```

Comment:
Same here and elsewhere below.  Leaving in obsolete code commented out adds clutter and may cause confusion.